core/geometry/src/geometry3d/Ray3d.ts

```diff
@@ -338,4 +338,27 @@ export class Ray3d implements BeJSONFunctions {
     pair.approachType = pairType;
     return pair;
   }
+
+  /**
+   * Return a ray whose ray.origin is interpolated, and ray.direction is the vector between points with a
+   * scale factor applied.
+   * @param fraction fractional position between points.
+   * @param pt2 endpoint of interpolation
```

Comment:
Should include @param pt1 start point of interpolation?

Reply:
thanks